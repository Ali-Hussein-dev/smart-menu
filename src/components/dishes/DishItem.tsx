import { GetDishes, Price, SvgIcon } from '../leaf-node'
import { useDisclosure } from '@chakra-ui/react'
import {
  Drawer,
  DrawerBody,
  DrawerHeader,
  DrawerOverlay,
  DrawerContent,
} from '@chakra-ui/react'
import { LongText } from '../leaf-node'

interface Props extends Menu.Item, Menu.DishItem {}
//=======================
export const DishItem: React.FC<{ item: Props }> = ({ item }) => {
  // hooks
  const { isOpen, onClose, onToggle } = useDisclosure()
  //--------------------------------------
  // functions
  //--------------------------------------
  return (
    <div className="flex justify-between mb-4 text-blueGray-600">
      <div className="">
        <div className="flex items-center gap-x-1">
          {item.info?.eatStyle?.infoList.filter((o) => o.label === 'Vegan')
            .length === 1 && (
            <SvgIcon size="w-6 h-6">
              <GetDishes iconName="vegan" color="bg-green-500" />
            </SvgIcon>
          )}
          <span className="leading-tighter">{item.name}</span>
        </div>
        <LongText>{item.description}</LongText>
        {item.dressing && (
          <span className="flex items-center gap-x-1 text-blueGray-500">
            <SvgIcon size="w-10 h-10 text-blue-300">
              <GetDishes iconName="dressing" />
            </SvgIcon>
            {item.dressing}
          </span>
        )}
      </div>
      <Price
        isInfo={true}
        price={item.price}
        isOpen={isOpen}
        onToggle={onToggle}
      />
      <Drawer placement={'bottom'} isOpen={isOpen} onClose={onClose} size="md">
        <DrawerOverlay>
          <DrawerContent>
            <DrawerHeader borderBottomWidth="1px">{item.name}</DrawerHeader>
            <DrawerBody>
              <h2 className="font-semibold">{item.info?.allergen?.header}</h2>
            </DrawerBody>
          </DrawerContent>
        </DrawerOverlay>
      </Drawer>
    </div>
  )
}
