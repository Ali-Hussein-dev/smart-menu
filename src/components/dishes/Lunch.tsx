import { withinTimeRange } from 'src/utils'
import dayjs from 'dayjs'
import { Ctx } from 'src/context'
import * as React from 'react'
// import { MdDateRange, MdAccessTime } from 'react-icons/md'
// import { IoPricetagsOutline } from 'react-icons/io5'
//=======================
export const Lunch: React.FC<{ currentTime: number }> = ({ currentTime }) => {
  // hooks
  const { state, dispatch } = React.useContext(Ctx)
  //--------------------------------------
  // functions
  React.useEffect(() => {
    if (
      state.meta.timeRanges.lunch.unavailableDays.filter((num) => {
        num === dayjs().get('d')
      }).length === 1
    ) {
      dispatch({ type: 'offLunch' })
    } else if (
      withinTimeRange(
        currentTime,
        //------------------make lunch available 3hrs earlier
        state.meta.timeRanges.lunch.start - 180,
        state.meta.timeRanges.lunch.end
      )
    ) {
      dispatch({ type: 'onLunch' })
    } else {
      dispatch({ type: 'offLunch' })
    }
  }, [currentTime])
  //--------------------------------------
  switch (state.meta.isLunch) {
    case true:
      return (
        <section className="text-blueGray-600">
          <div className="flex flex-col pl-2 mb-2 gap-y-2">
            <h2 className="font-bold text-center">Mittagessen</h2>
            {/* <span className="flex items-center gap-x-2">
              <MdDateRange size="20" /> 12.3.2021 - 20.3.2021 | Montag bis
              Freitag
            </span>
            <span className="flex items-center gap-x-2">
              <MdAccessTime size="20" />
              11:30-17:00 Uhr
            </span>
            <span className="flex items-center gap-x-2">
              <IoPricetagsOutline size="20" />
              8.80â‚¬
            </span> */}
          </div>
          <div className="px-2">
            <img
              className="w-full mx-auto mb-4 object-fill rounded-lg"
              src="https://www.foodiesfeed.com/wp-content/uploads/2021/01/fried-salmon-with-sweet-soy-sauce-in-a-korean-restaurant.jpg"
            />
          </div>
        </section>
      )
    default:
      return null
  }
}
