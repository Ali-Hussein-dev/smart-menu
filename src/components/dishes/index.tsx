export * from './Lunch'
export * from './DishItem'
