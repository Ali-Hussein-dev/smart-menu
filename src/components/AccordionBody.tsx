import DishItem from './DishItem'

//--------------------------------------
const ExtraItem: React.FC<{
  extra: {
    list: { name: string; price: number; label: string }[]
    header4: string
  }
}> = ({ extra }) => {
  // hooks
  //--------------------------------------
  // functions
  //--------------------------------------
  return (
    <div className="p-2 rounded-sm text-blueGray-600 bg-amber-100">
      <div className="flex">
        <h3 className="text-sm font-semibold md:text-lg text-blueGray-500">
          {extra.header4}{' '}
        </h3>
      </div>
      {extra.list.map((o) => (
        <div
          key={o.name}
          className="flex items-start justify-between gap-x-1 md:text-lg"
        >
          <span className="mb-2 text-sm font-normal md:text-lg ">
            {o.name}{' '}
          </span>
          <div className="flex items-center gap-x-1">
            <span className="text-sm font-light tracking-tight md:text-lg">
              {o.label}{' '}
            </span>
            <span className="font-normal">{o.price.toFixed(2)} </span>
          </div>
        </div>
      ))}
    </div>
  )
}
//=======================
const AccordionBody: React.FC<{
  extra?: any
  ImgSrc: string
  list: any[]
  id: number
}> = ({ extra, list: list, ImgSrc, id }) => {
  // hooks
  //--------------------------------------
  // functions
  //--------------------------------------
  return (
    <div data-testid={id.toString()} className="px-2">
      <img className="" src={ImgSrc} />
      {list.map((o) => (
        <DishItem
          key={o.name}
          name={o.name}
          description={o.description}
          price={o.price}
        />
      ))}
      {extra && <ExtraItem extra={extra} />}
    </div>
  )
}
export default AccordionBody
