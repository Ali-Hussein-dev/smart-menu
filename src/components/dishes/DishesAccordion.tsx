import {
  Accordion,
  AccordionItem,
  AccordionPanel,
  AccordionButton,
} from '@chakra-ui/react'
import { DishItem } from './'
import { CC, GetDishes, MenuTop, Price } from '../leaf-node'
import * as React from 'react'
import { Lunch } from './Lunch'
import dayjs from 'dayjs'

//=======================
const DishesAccordion: React.FC<{
  dishes: [Menu.Header, Menu.List<Menu.Item>][]
}> = ({ dishes: list }) => {
  // hooks
  //--------------------------------------
  // functions
  //--------------------------------------
  return (
    <>
      <Lunch currentTime={dayjs().get('h') * 60 + dayjs().get('m')} />
      <Accordion allowToggle>
        {list.map((o, i) => (
          <AccordionItem key={i}>
            <AccordionButton
              _focus={{ bg: 'gray.400', outline: 'none' }}
              _expanded={{ bg: 'gray.200', boxShadow: 'md' }}
            >
              <CC isTrue={o[0].iconName}>
                <MenuTop
                  header={o[0].header}
                  Component={<GetDishes iconName={o[0].iconName} />}
                />
              </CC>
            </AccordionButton>
            <AccordionPanel>
              {o[1].list.map((arr, i) => (
                <div key={i}>
                  <CC isTrue={arr[0]?.header}>
                    <h3 className="mb-5 font-semibold text-center">
                      {arr[0].header}
                    </h3>
                  </CC>
                  <MenuTop
                    header={arr[0].header}
                    assest={{ imgSrc: arr[0].imgSrc }}
                  />
                  {arr[1].itemsList.map((oo, ii) => (
                    <DishItem key={ii} item={oo} />
                  ))}
                  <CC isTrue={arr[1].addon}>
                    <div className="addon-list">
                      <h4 className="mb-3 font-semibold">
                        {arr[1].addon?.header}
                      </h4>
                      {arr[1].addon?.addonList.map((ooo, iii) => (
                        <div
                          key={iii}
                          className="flex justify-between mb-2 text-blueGray-600"
                        >
                          <span>{ooo.name}</span>
                          <Price price={ooo.price} />
                        </div>
                      ))}
                    </div>
                  </CC>
                </div>
              ))}
            </AccordionPanel>
          </AccordionItem>
        ))}
      </Accordion>
    </>
  )
}
export default DishesAccordion
