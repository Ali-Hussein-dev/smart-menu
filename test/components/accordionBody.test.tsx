import { render } from '@testing-library/react'
import AccordionBody from '../../src/components/AccordionBody'
import dishes from '../../database/dishes/dishes'

describe('AccordionBody', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <AccordionBody ImgSrc={dishes[0].ImgSrc} list={dishes[0].list} id={2} />
    )
    expect(container).toMatchSnapshot()
  })
  test('should match snapshot when extra is true', () => {
    const { container } = render(
      <AccordionBody
        ImgSrc={dishes[1].ImgSrc}
        list={dishes[1].list}
        extra={dishes[1].extra}
        id={2}
      />
    )
    expect(container).toMatchSnapshot()
  })
})
