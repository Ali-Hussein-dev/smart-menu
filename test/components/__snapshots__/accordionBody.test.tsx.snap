// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccordionBody should match snapshot 1`] = `
<div>
  <div
    class="px-2"
    data-testid="2"
  >
    <img
      class=""
      src=""
    />
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Chili con Carne vom Galloway Rind
        </span>
        <p
          class=""
        >
          Mexikanischer Hackfleisch-Bohneneintopf mit Sauerrahm und frischem Brot
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          11.50
        </span>
        <button
          aria-label="info-11.5"
          class="p-1"
          id="info-11.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Sweet Potato Fries and Truffle
        </span>
        <p
          class=""
        >
          Pommes frites von der Süßkartoffel mit hausgemachter Trüffelmayonnaise
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          7.50
        </span>
        <button
          aria-label="info-7.5"
          class="p-1"
          id="info-7.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Veganes Chili sin Carne
        </span>
        <p
          class=""
        >
          Mexikanischer Soja-Bohneneintopf mit frischem Brot
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          10.50
        </span>
        <button
          aria-label="info-10.5"
          class="p-1"
          id="info-10.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Bacon Cheese Fries
        </span>
        <p
          class=""
        >
          Pommes frites mit herzhafter Käsesauce und knusprig gebratenen Baconwürfeln vom Strohschwein
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          5.50
        </span>
        <button
          aria-label="info-5.5"
          class="p-1"
          id="info-5.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Chili Cheese Fries
        </span>
        <p
          class=""
        >
          Pommes frites mit pikantem Chili con Carne, Jalapeños und leckerer Cheddarsauce
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          9.00
        </span>
        <button
          aria-label="info-9"
          class="p-1"
          id="info-9"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Veganes Sandwich mit einer herzhaften Algencreme 'Tuna-Style' 
        </span>
        <p
          class=""
        >
          Creme aus Algen und Kichererbsen mit frischer Paprika, roten Zwiebeln, Mais und knusprigen Kartoffelchips auf geröstetem Weißbrot mit Gurken Tomaten und Eisbergsalat dazu Pommes Frite
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          12.50
        </span>
        <button
          aria-label="info-12.5"
          class="p-1"
          id="info-12.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Carpaccio nach Cipranis Originalrezept
        </span>
        <p
          class=""
        >
          vom Rinderfilet, mariniert mit Trüffelöl, Rauke Salat, Ciprianisauce und frischen Hobel von Italienischem Hartkäse 
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          13.50
        </span>
        <button
          aria-label="info-13.5"
          class="p-1"
          id="info-13.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`AccordionBody should match snapshot when extra is true 1`] = `
<div>
  <div
    class="px-2"
    data-testid="2"
  >
    <img
      class=""
      src="/trude/salat.jpg"
    />
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Kleiner Gartensalat
        </span>
        <p
          class=""
        >
          Bunte Blattsalate mit Tomate, Gurke, Möhre, Chicorée und frischen Kräutern
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          7.00
        </span>
        <button
          aria-label="info-7"
          class="p-1"
          id="info-7"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Großer Gartensalat
        </span>
        <p
          class=""
        >
          Bunte Blattsalate mit Tomate, Gurke, Paprika, frischen Kräutern und gebackenen Flammkuchenchips
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          9.50
        </span>
        <button
          aria-label="info-9.5"
          class="p-1"
          id="info-9.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Caesar's Salat
        </span>
        <p
          class=""
        >
          Knackiger Römersalat und Kirschtomaten in einem mit feinen Kräutern und zerstoßenem Pfeffer  mit Croûtons und Parmesanspänen
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          10.50
        </span>
        <button
          aria-label="info-10.5"
          class="p-1"
          id="info-10.5"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Rucola Melonoensalat
        </span>
        <p
          class=""
        >
          Frischer Rucola und saftige Honigmelone mit mariniertem Hirtenkäse
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          13.00
        </span>
        <button
          aria-label="info-13"
          class="p-1"
          id="info-13"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="flex justify-between bg-blueGray-400"
    >
      <div
        class=""
      >
        <span
          class="font-semibold"
        >
          Kerniger Nusssalat
        </span>
        <p
          class=""
        >
          Bunte Blattsalate, Apfelchips, Walnüsse, Cashewkerne, Croûtons mit Möhren und Kirschtomaten
        </p>
      </div>
      <div
        class="flex flex-col items-center"
      >
         
        <span>
          10.00
        </span>
        <button
          aria-label="info-10"
          class="p-1"
          id="info-10"
          role="button"
        >
          <svg
            fill="currentColor"
            height="15"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="15"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M7 4.75c0-0.412 0.338-0.75 0.75-0.75h0.5c0.412 0 0.75 0.338 0.75 0.75v0.5c0 0.412-0.338 0.75-0.75 0.75h-0.5c-0.412 0-0.75-0.338-0.75-0.75v-0.5z"
            />
            <path
              d="M10 12h-4v-1h1v-3h-1v-1h3v4h1z"
            />
            <path
              d="M8 0c-4.418 0-8 3.582-8 8s3.582 8 8 8 8-3.582 8-8-3.582-8-8-8zM8 14.5c-3.59 0-6.5-2.91-6.5-6.5s2.91-6.5 6.5-6.5 6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="p-2 rounded-sm text-blueGray-600 bg-amber-100"
    >
      <div
        class="flex"
      >
        <h3
          class="text-sm font-semibold md:text-lg text-blueGray-500"
        >
          Beilage zum Salat
           
        </h3>
      </div>
      <div
        class="flex items-start justify-between gap-x-1 md:text-lg"
      >
        <span
          class="mb-2 text-sm font-normal md:text-lg "
        >
          Kleine gegrillte Hähnchenbrust 120g vom Kikok
           
        </span>
        <div
          class="flex items-center gap-x-1"
        >
          <span
            class="text-sm font-light tracking-tight md:text-lg"
          >
             
          </span>
          <span
            class="font-normal"
          >
            5.50
             
          </span>
        </div>
      </div>
      <div
        class="flex items-start justify-between gap-x-1 md:text-lg"
      >
        <span
          class="mb-2 text-sm font-normal md:text-lg "
        >
          Gebratene Champignons und Kräuterseitlinge
           
        </span>
        <div
          class="flex items-center gap-x-1"
        >
          <span
            class="text-sm font-light tracking-tight md:text-lg"
          >
             
          </span>
          <span
            class="font-normal"
          >
            3.50
             
          </span>
        </div>
      </div>
      <div
        class="flex items-start justify-between gap-x-1 md:text-lg"
      >
        <span
          class="mb-2 text-sm font-normal md:text-lg "
        >
          Parmeasan
           
        </span>
        <div
          class="flex items-center gap-x-1"
        >
          <span
            class="text-sm font-light tracking-tight md:text-lg"
          >
             
          </span>
          <span
            class="font-normal"
          >
            2.50
             
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
